app/javascript/src/utils/date.js

```diff
@@ -0,0 +1,5 @@
+import dayjs from "dayjs";
+
+export const calculateTimeDifference = date => dayjs(date).fromNow();
+
+export const dayTimeFormatter = date => dayjs(date).format("dddd, hh:mmA");
```

Comment:
https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Keep%20the%20files%20in%20closest%20scope

Reply:
I added a date.js file to src/utils because I plan to use dayjs functions in the Contacts page as well. 
Should I create separate util files for both in their closest scope?

Reply:
In that case, you can keep this within this file.